stdlib/3.3/xml/etree/ElementTree.pyi

```diff
@@ -80,6 +80,7 @@ class _ListDataStream(io.BufferedIOBase):
     def __init__(self, lst) -> None: ...
     def writable(self) -> bool: ...
     def seekable(self) -> bool: ...
+    @no_type_check  # not same signature as parent
     def write(self, b: str) -> None: ...
```

Comment:
I recommend using `# type: ignore` here instead of `@no_type_check`. The latter means that the annotations aren't even in mypy's language; the former just suppresses the complaint about the type mismatch.

Though maybe there's a different solution possible altogether? Having to suppress this smells a bit...